src/Storage/Storage/Storage.csproj

```diff
@@ -14,9 +14,9 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.Azure.Cosmos.Table" Version="0.10.1-preview" />
     <PackageReference Include="System.Security.Permissions" Version="4.5.0" />
-    <PackageReference Include="Microsoft.Azure.Storage.Blob" Version="10.0.3" />
-    <PackageReference Include="Microsoft.Azure.Storage.File" Version="10.0.3" />
-    <PackageReference Include="Microsoft.Azure.Storage.Queue" Version="10.0.3" />
+    <PackageReference Include="Microsoft.Azure.Storage.Blob" Version="11.0.0" />
+    <PackageReference Include="Microsoft.Azure.Storage.File" Version="11.0.0" />
```

Comment:
This is going to cause problems with the DataMovement library - in WIndows PowerSHell, it will attempt to load (and fail) the previous version of the storage file and storage blob libraries.

Reply:
As above comments, I have upgraded the private signed DMlib to new build with this PR. So this won't take problem.

Reply:
Got it.  We will do a sign build and run soem tests